content/800-data-platform/300-data-proxy/900-considerations-limitations.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: 'Considerations and limitations'
+metaTitle: 'Considerations'
+metaDescription: 'When you use Data Proxy for applications in serverless and edge runtimes, you should bear in mind a few considerations and learn about the current limitations.'
+tocDepth: 2
+---
+
+## Considerations
+
+### Connection to databases hosted with a provider different from AWS
+
+It is possible to connect to a database hosted on any provider, but it needs to be accessible from the public internet. For best results and response times, you should configure the proxy in a region that is the same or as close as possible to the database.
+
+We are looking to expand the number of providers and regions in the future. If a region or provider is not yet supported, you can [submit a request](https://pris.ly/proxy-new-region). This will help us prioritize providers and regions to add next.
+
+### Recommended concurrent connections
+
+We recommend that you use a database that supports at least 20 concurrent connections.
```

Comment:
Would it be worth adding a note or link here about tweaking the number database connections in the proxy, e.g. adjusting connection pool size?